src/Bridge/Doctrine/Orm/Filter/ExistsFilter.php

```diff
@@ -44,33 +43,9 @@ class ExistsFilter extends AbstractContextAwareFilter implements ExistsFilterInt
 
     /**
      * @param RequestStack|null $requestStack No prefix to prevent autowiring of this deprecated property
-     * @param mixed|null        $properties
      */
-    public function __construct(ManagerRegistry $managerRegistry, $requestStack = null, LoggerInterface $logger = null, /* string $existsParameterName = self::QUERY_PARAMETER_KEY, array*/ $properties = null)
+    public function __construct(ManagerRegistry $managerRegistry, $requestStack = null, LoggerInterface $logger = null, array $properties = null, string $existsParameterName = self::QUERY_PARAMETER_KEY)
```

Comment:
The same strategy from #2664 should be applied here.

Reply:
is it really worth it ? This argument is truly optional

Reply:
This comment was about changing the typehint of `$requestStack` as explained off, it has been done